src/components/login-modal/LoginModal.vue

```diff
@@ -28,7 +28,7 @@
             <div v-if="isSignIn" class="sign-in-wrapper">
               <div class="form-group">
                 <label>{{ $t('generic.email') }}</label>
-                <input type="email" class="form-control" name="email" id="email" v-model="form.email">
+                <input type="email" class="form-control" name="email" id="email" v-model="form.email" placeholder="auth.emailPlaceholder">
```

Comment:
Are you sure you want to have the text `auth.emailPlacholder`? 

There are 3 options right now in vuejs:

1. when you just simply show a text/string

`placeholder="somettexthere"`

2. when you take a value from somewhere which contains a text/string:

`:placeholder="someValuehere`

3. when you take a string from the translation string, which I think this is what you need here:

`:placeholder="$t('auth.emailPlaceholder')"`